### PR TITLE
[MIG][15.0] portal: migration script

### PR DESCRIPTION
**Pull request by sang250399**
This PR
=======
Migration script

### CODE REVIEW

File: openupgrade_scripts/scripts/portal/15.0.1.0/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---Models in module 'portal'---
+---Fields in module 'portal'---
+portal       / adyen.account            / website_message_ids (one2many): DEL relation: mail.message
+portal       / crm.team.member          / website_message_ids (one2many): NEW relation: mail.message
+portal       / uom.category             / website_message_ids (one2many): NEW relation: mail.message
```
Comment (legalsylvain) [inline openupgrade_scripts/scripts/portal/15.0.1.0/upgrade_analysis_work.txt]: I don't understand why this file contains this line. 
The original one doesn't contains line 5 https://github.com/OCA/OpenUpgrade/blob/15.0/openupgrade_scripts/scripts/portal/15.0.1.0/upgrade_analysis.txt

**Review by legalsylvain — APPROVED** [review]
a minor question. 

Otherwise, LGTM.

**Comment by MiquelRForgeFlow** [comment]
Merged in https://github.com/OCA/OpenUpgrade/pull/3464.

**Pull request closed without merging**